tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/MatterProvision.cs

```diff
@@ -345,6 +354,7 @@ public MatterStampedDetails GetStampedProperties(MatterVM matterVM)
                         MatterObject = new Matter()
                         {
                             Id = GetStampPropertyValue(stampedPropertyValues, matterSettings.StampedPropertyMatterID),
+                            MatterGuid = GetStampPropertyValue(stampedPropertyValues, matterSettings.StampedPropertyMatterGUID),

```

Comment:
When sending the stamped properties, also sending MatterGuid and this property will be used in ExternalSharing functionality